codex/rest/api.nim

```diff
@@ -453,8 +456,11 @@ proc initPurchasingApi(node: CodexNodeRef, router: var RestRouter) =
         without params =? StorageRequestParams.fromJson(body), error:
           return RestApiResponse.error(Http400, error.msg, headers = headers)
 
-        let nodes = params.nodes |? 1
-        let tolerance = params.tolerance |? 0
+        let nodes = params.nodes |? 2
+        let tolerance = params.tolerance |? 1
+
+        if tolerance == 0:
+          return RestApiResponse.error(Http400, "Tolerance needs to be bigger then zero", headers = headers)
```

Comment:
It feels like this should be in its own PR. At the very least let's add some comments around it (maybe just copy pasta from the discord discussion).